whisper/src/point.rs

```diff
@@ -1,5 +1,7 @@
 use std::io;
 use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+use std::str::FromStr;
+use failure::Error;
 
 #[derive(Debug, Clone, Copy)]
 pub struct Point {
```

Comment:
This structure is already implemented.It's called 'Retention'

Reply:
this is not a retention. This is a Point